Cql/CodeGeneration.NET/CSharpSourceCodeWriter.cs

```diff
@@ -385,14 +277,6 @@ private IList<DirectedGraphNode> DetermineBuildOrder(DirectedGraph minimalGraph)
             return sorted;
         }
 
-        private string DefinitionCacheKeyForMethod(string methodName)
-        {
-            if (methodName[0] == '@')
-                return "__" + methodName.Substring(1);
-            else return "__" + methodName;
-        }
-        private string PrivateMethodNameFor(string methodName) => methodName + "_Value";
-
         private void WriteMemoizedInstanceMethod(string libraryName, TextWriter writer, int indentLevel,

```

Comment:
Rename to WriteMethod

Reply:
Tested the singleton approach and it produced similar results so either works